Assets/MRTK/Providers/LeapMotion/Definitions/LeapControllerOrientation.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+namespace Microsoft.MixedReality.Toolkit.LeapMotion.Input
+{
+    /// <summary>
+    /// The location of the leap motion controller. LeapControllerOrientation.Headset indicates the controller is mounted on a headset. 
```

Comment:
Since you've added the description to the enum values themselves, there's no reason to have the comments duplicated here (so I think this doccomment can be reduced to just:

The location of the leap motion controller

Reply:
Updated comment